Formula/vgmstream.rb

```diff
@@ -1,10 +1,11 @@
 class Vgmstream < Formula
   desc "Library for playing streamed audio formats from video games"
   homepage "https://hcs64.com/vgmstream.html"
-  url "https://github.com/kode54/vgmstream/archive/r1040.tar.gz"
-  version "r1040"
-  sha256 "0ff6534a4049b27b01caf209811b87b1bfe445f94e141a5fe601f2dae9d03c89"
+  url "https://github.com/losnoco/vgmstream/archive/r1050-2441-gd64c3872.tar.gz"
+  version "r1050-2441-gd64c3872"
+  sha256 "e1d7dfbf65b9b58ebb8c3c3792f8d7a04de55c03e5292e5ac68de5e60c65587b"
   head "https://github.com/kode54/vgmstream.git"
+  revision 1
```

Comment:
Why?

Reply:
I'm trying to read through the docs for the first time.  Thought if there are bottles for the old version, need new ones made from the source update I made.

Reply:
In my opinion this line should be deleted, because it’s not a revision. (And `revision` should go before `head` anyway.)

Reply:
Yea, just saw that in the build output.  I don't understand it yet, so I will.